### PR TITLE
Fix style of the Shopping Cart icon

### PR DESCRIPTION
**Pull request by olegrise**
@RV-Adrian or @ghislaineguerin could one of you please review. It seems like a combination of changes in the latest versions of the CH and Common Style broke the styling of the Shopping Cart. [Here](https://store-stage-2.risevision.com/) is how it looks like with the CH v1.10.9 and Common Style v1.3.70:

bad

And [here](https://store-stage-3.risevision.com/) is how it suppose to look:

good.

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@olegrise Can you stage with this updated CS branch and see how it works out? Thx!

File: src/sections/store-nav.scss [hunk 1]
```diff
@@ -65,7 +65,12 @@ body.in-rva-iframe {
 
 .navbar-right>li>a.shopping-cart-button {
   font-size: 24px;
+  font-size: 24px;
```
Comment (olegrise) [inline src/sections/store-nav.scss]: @RV-Adrian do we need `font-size: 24px;` twice? :)

**Comment by olegrise** [comment]
@RV-Adrian staged at https://store-stage-0.risevision.com/

File: src/sections/store-nav.scss (hunk 1 above)
Comment (RV-Adrian) [inline src/sections/store-nav.scss]: @olegrise Definitely not :) Adjusted

**Comment by olegrise** [comment]
Thank you @RV-Adrian !!!

**Pull request merged**